files/build_templates/docker_image_ctl.j2

```diff
@@ -368,6 +368,7 @@ start() {
         fi
 
         {%- if docker_container_name == "database" %}
+        NET="bridge"
```

Comment:
This setting was present earlier eg: 201911 release etc. It got removed in one of the PR's https://github.com/Azure/sonic-buildimage/pull/5283